apps/web/src/app/_components/companies/company-about.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import type { CompanyType } from "@cooper/db/schema";
+import { Card, CardContent, CardHeader, CardTitle } from "@cooper/ui/card";
+
+interface CompanyAboutProps {
+  className?: string;
+  companyObj: CompanyType | undefined;
+}
+
+export function CompanyAbout({ companyObj }: CompanyAboutProps) {
+  return (
+    <>
+      {" "}
+      <Card className="w-full max-w-lg rounded-xl outline outline-[1px] outline-[#474747]">
+        <CardHeader className="border-b border-[#474747] bg-[#F7F7F7] pb-3">
```

Comment:
I mention this below too:

Let's remove pb-3 and replace it with h-6 flex justify-center.

The way I got h-6 was by using dev mode on the design to see it was 24px. Then I went to https://v3.tailwindcss.com/docs/size and found that h-6 corresponds to 24px.